### ISSUE TITLE
Shift+tab traps focus inside WinForms WebView2 control when wrapped in a ContainerControl

### ISSUE THREAD

**Issue by MackinnonBuck** [MicrosoftEdge/WebView2Feedback#2835]
**Description**
When a WinForms `WebView2` control is a child element of a `ContainerControl`, shift+tab navigation cannot move focus out of the WebView. While focus cannot escape the WebView in the reverse direction, it _can_ escape the WebView in the forward direction.

This bug affects the `BlazorWebView` WinForms control, which extends `ContainerControl` and wraps the `WebView2` control.

**Version**
SDK: 1.0.1369-prerelease
Runtime: Stable 105.0.1343.53
Framework: WinForms
OS: Windows 11 Enterprise Version 22H2

**Repro Steps**
1. Create a new Windows Forms App (.NET 7) and add a reference to `Microsoft.Web.WebView2`.
2. Replace the code in the `Form1` constructor with the following:
```csharp
InitializeComponent();

var button1 = new Button { Text = "button1", TabIndex = 0 };
var webViewContainer = new ContainerControl { Width = 220, Height = 40, Top = 30, TabIndex = 1 };
var webView = new Microsoft.Web.WebView2.WinForms.WebView2 { Dock = DockStyle.Fill };

webViewContainer.Controls.Add(webView);

Controls.Add(button1);
Controls.Add(webViewContainer);

_ = InitializeWebViewAsync();

async Task InitializeWebViewAsync()
{
    await webView.EnsureCoreWebView2Async();
    webView.NavigateToString("""
        <!DOCTYPE html>
        <head>
        </head>
        <body>
            <button type="button">button2</button>
            <button type="button">button3</button>
            <button type="button">button4</button>
        </body>
        """);
}
```
3. Run the application.
4. Repeatedly press tab, observing that the focus correctly enters and exits the WebView.
5. Repeatedly press shift+tab, observing that focus gets trapped inside the WebView.

_Expected behavior:_ When the first HTML element in the WebView has focus and shift+tab is pressed, the Windows Forms control previous to the `WebView2` control should receive focus.

_Actual behavior:_ When the first HTML element in the WebView has focus, shift+tabbing results in the last element in the WebView receiving focus.

**Screen recording**

https://user-images.githubusercontent.com/10456961/193145580-1a5e3bf5-0229-4b0b-aeda-9baa99466251.mp4

**Additional context**

https://github.com/dotnet/maui/issues/8293

[AB#43460910](https://microsoft.visualstudio.com/90b2a23c-cab8-4e7c-90e7-a977f32c1f5d/_workitems/edit/43460910)

**Comment by victorthoang** [comment]
hello @MackinnonBuck, 

I've assigned your query to one of our devs to investigate the bug. Thanks and we shall get back to you.

**Comment by ElyssaJyu** [comment]
Hi @MackinnonBuck, thanks for your report, it seems a bug exists, I will track it internally.
If this blocks your work, as a current alternative, you can use `Controls.Add(webView)`, instead of using `ContrainerControl` to wrapper webview2

**Comment by ShaunLoganOracle** [comment]
This issue is shown as fixed in the [release notes](https://learn.microsoft.com/en-us/microsoft-edge/webview2/release-notes?tabs=dotnetcsharp#101777-prerelease) for 1.0.1777-prerelease.  I was hopeful that the same fix might have helped with #951 , but that issue still reproduces.
